### PR TITLE
Add week 7 blog post

### PR DESCRIPTION
**Pull request by haroldtreen**
Week 7 blog post is almost done!

Happily will accept comments/feedback @drigberg & @yuliaju. ❤️

### CODE REVIEW

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+title: "Week[6] @ The Recurse Center 🏁"
+comments: true
+categories: tech recurse
+---
+
+A week with the Winter 1s complete! Time for another review.
+
+### Overview
+
+I've started feeling much more focused at RC. The change in people has really helped this. Exited for what the next 5 weeks brings!
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md]: "excited" :)

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+### Emotional 😊
+
+#### **Blogging is causing me grief...**
+
+Once again getting deep into a new week and freaking out about getting a blog post done.
+
+Is it going to be an accurate portrayal of what I did last week? How was I feeling? What did I do? How do I summarize all that?
+
+It's hard to blog in the present about how I felt in the past... at least without it feeling forced and unnatural. My new approach is going to be blogging about my current feels and relating those feels to the past week.
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md:22]: GAAHHH TOO REAL

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 1 above)
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md]: "excited" and "bring" :) Stop me if you don't want spelling/grammar comments!!!!!

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+There's a group of people here who I've known for the past 7 weeks.
+There's another group of people who are a blank slate. I can be whoever I want to be around them. They have no expectations for the space.
+
+As time goes on, I gain more familiarity with the environment and people around me. When I first got to New York everything was new and I struggled to take it all in. Now that I've adjusted I can do a better job focusing on different interests.
+
+#### **Election feels**
+
+Last week was the election and a large number of people at RC were very effected by the results.
+
+I was also shocked, but not being a US resident has buffered me from the full feels. Was unsure how to best support my fellow RCers, and still not sure.
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md:37]: I think for a lot of us, just not mentioning it is enough--having it be everywhere all the time is a lot. I love that I can escape when I'm at RC :)

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+> "If I spend all my time doing social stuff, am I becoming a better programmer?"
+
+I've started to realize that "Becoming a better programmer" is more than lines of code, github activity and pull requests.
+
+A good programmer makes others exited about their projects.  
+A good programmer teaches complex concepts in easy to understand ways.  
+A good programmer gives back to the communities they are a part of.  
+
+I do feel like a better programmer... just not in the way I had originally envisioned.
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md:105]: yesssssssssss

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+But in the past 6 weeks I've done very little of this. Instead I've:
+- Paired with a ton of people.
+- Pondered how I learn, and the systems that support that.
+- Dived into existential questions about what I want to do with software.
+
+I started feeling really guilty about that.
+
+> RC: "So why do you want to come to RC?"
+
+> Harold: "I want to come to RC because I have a project I'm passionated about, but I'm currently building it in hostels. I need a better environment to build cool things."
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md]: *passionate

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+#### **Job prospects that are exciting**
+
+This week while perusing my news feed, I came across a job posting for [Ladies Learning Code](http://ladieslearningcode.com/) in Vancouver. Picturing the job opportunity to work for a diverse tech community in Vancouver got me really excited.
+
+Then I starting judging:
+
+> "But Harold, you wouldn't earn big cash $s!"
+
+> "Who's going to know Ladies Learning Code? You'll stop looking like a software engineer!"
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md:133]: name recognition = _blegh_
Ladies Learning Code sounds like the best thing

File: _posts/2016-11-13-recurse-center-week-7.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+#### **Looking forward to exercise**
+
+I never got around to getting into an exercise schedule. I'm not going to make excuses, it hasn't felt important.
+
+Spending time with people, learning to code and being at RC has been the focus. I do enjoy daydreaming about settling down and being able to get in shape again.
+
+#### **Macaroni is delicious**
+
+This week I randomly got into a conversation about Macaroni and Cheese. I had forgotten how simple and delicious it is to make, so I made a giant pot this week.
```
Comment (drigberg) [inline _posts/2016-11-13-recurse-center-week-7.md]: 👍

**Review by drigberg — COMMENTED** [review]
w00t!!!!

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 2 above)
Comment (haroldtreen) [reply _posts/2016-11-13-recurse-center-week-7.md:22]: WE BE THE REALEST

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 1 above)
Comment (haroldtreen) [reply _posts/2016-11-13-recurse-center-week-7.md]: I'm exiting for the next 5 weeks brings is what I thought I was trying to say.

Good catch! 👍

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 3 above)
Comment (haroldtreen) [reply _posts/2016-11-13-recurse-center-week-7.md:37]: Cool! I heard the politics at RC disclaimer yesterday and it's nice to see that validated.

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 5 above)
Comment (haroldtreen) [reply _posts/2016-11-13-recurse-center-week-7.md]: *passionating 😉

File: _posts/2016-11-13-recurse-center-week-7.md (hunk 1 above)
Comment (haroldtreen) [reply _posts/2016-11-13-recurse-center-week-7.md]: (I keep spelling "exercise" with a "C" - "Excercise". And then I think "NO! NO C AFTER X"... and then I spell things like "exited"...)

**Comment by haroldtreen** [comment]
WOO! No more passionated-ing!

Thanks @drigberg 

MERGEY-MERGING

**Pull request merged**